docs/specs/xref.yml

```diff
@@ -20,8 +20,10 @@ outputs:
     ["info","at-uid-not-found","Cannot find uid 'a' using xref '@a'","docs/a.md"]
 ---
 # Restore xref map from configured url and use the map for resolving
+# hostname should be removed if docset sharing the same one
 inputs:
   docfx.yml: |
+    baseUrl: docs.microsoft.com
```

Comment:
`baseUrl` contains `https://`, so this should be 'https://docs.microsoft.com'

Reply:
ok